packages/components/src/DateField/DateField.tsx

```diff
@@ -97,7 +97,7 @@ export const DateField = ({
           if (
             val.length > 4 ||
             Number(val) > MAX_YEAR_NUMBER ||
-            (val.length === 2 && Number(val) < 19)
+            (val.length === 2 && Number(val) < 10)
```

Comment:
If we are allowing any year value to be put in as 4 digits, then we shouldn't be supporting 2 digit years as that would need us to make an assumption about the century